visidata/sheets.py

```diff
@@ -1210,13 +1210,8 @@ def _async_deepcopy(newlist, oldlist):
 BaseSheet.addCommand(None, 'rename-sheet', 'sheet.name = input("rename sheet to: ", value=sheet.name)', 'Rename current sheet')
 
 
-@Column.cached_property
-def _format(col):
-    return dict(val.split('=', maxsplit=1) for val in col.fmtstr.split())
```

Comment:
I can return this here. This got moved to `_formatdict` in `columns.py`. I removed the `cached_property`, but I can return that.